tests/test_configuration.py

```diff
@@ -897,3 +898,27 @@ def test_pairlist_resolving_fallback(mocker):
     assert config['pairs'] == ['ETH/BTC', 'XRP/BTC']
     assert config['exchange']['name'] == 'binance'
     assert config['datadir'] == str(Path.cwd() / "user_data/data/binance")
+
+
+@pytest.mark.parametrize("setting", [
+        ("ask_strategy", "use_sell_signal", True,
+         "experimental", "use_sell_signal", False),
+        ("ask_strategy", "sell_profit_only", False,
+         "experimental", "sell_profit_only", True),
+        ("ask_strategy", "ignore_roi_if_buy_signal", False,
+         "experimental", "ignore_roi_if_buy_signal", True),
+    ])
+def test_deprecated_experimental(mocker, default_conf, setting, caplog):
+    patched_configuration_load_config_file(mocker, default_conf)
+
+    default_conf[setting[3]] = {}
+    default_conf[setting[0]][setting[1]] = setting[2]
+    default_conf[setting[3]][setting[4]] = setting[5]
```

Comment:
maybe a comment "#set old value" and "#set new value" (for the above e lines) would benefit readability
had to think twice what what number actually is ...

Reply:
Agree, I also thought about the elements when I returned back to this test...